botCommands/leaderboard.js

```diff
@@ -10,30 +10,31 @@ const leaderboard = {
         .split(' ')
         .find((word) => word.includes('start='));
       let start = sEquals ? sEquals.replace('start=', '') : 1;
-      start = Math.max(start, 1);
+      start = Number.isNaN(Number(start)) ? 1 : Math.max(start, 1);
 
       const nEquals = content.split(' ').find((word) => word.includes('n='));
       let length = nEquals ? nEquals.replace('n=', '') : 5;
+      length = Number.isNaN(Number(length)) ? 5 : length;
```

Comment:
Great catch, thanks for adding these checks.

Reply:
Anytime!